Add_user.css

```diff
@@ -104,44 +128,76 @@ ul li:hover ul{
 	width:100%
 }
 
+/* Set up the footer */
 .footer{
-	background-color: white;
-	grid-column-start: 1;
+	background-color: #0C59CF;
 	grid-column-end: span 4;
+	padding-top:25px;
 	text-align: center;
-	height:35px;
-	color:black;
-	padding-top:20px;
+	color:white;
+	height:50px;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:300;
 }
 
+/* Re position of text in the div */
 h3{
 	margin-left:130px;
 	margin-top:30px;
 	
 }
 
+/* Re position of text in the div and change font-size, color, weight */
 p{
 	margin-left:130px;
 	margin-top:250px;
 	font-size:24px;
-	color:black;
-	font-weight: bold;
+	color:#0C59CF;
+	font-family: 'Source Sans Pro', sans-serif;
+	font-weight:600;
 }
 
 #us{
 	height:30px;
 	width: 240px;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:300;
 }
 
 #pa{
 	height:30px;
 	width: 240px;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:300;
 }
 
+/* Set up the login buttom*/
 #login{
 	height:25px;
 	width: 80px;
-	color:black;
-	background-color:white;
+	color:white;
+	background-color:#0C59CF;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:300;
 }
 
+/* Set up the text display correct*/
+.Connected{
+	font-size:18px;
+	color:green;
+	font-weight: bold;
+	position: absolute;
+  	top: 240px;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:300;
+}
+
+/* Set up the text display error*/
+.error{
```

Comment:
I set up this so that when the user create a invalid account than it will display to show user that they are fail to create an account. So they can tried again. This is help they have a better experience. Because if there wasn't any indicator they might think they was successfully create an account but they can't able to login and  this can result stress and waste of time.